clients/corral/corral.go

```diff
@@ -68,9 +68,9 @@ func SetCustomRepo(repo string) error {
 	return nil
 }
 
-// CreateCorral creates a corral taking the corral name, the package path, and a debug set so if someone wants to view the
+// CreateCorralAndWait creates a corral taking the corral name, the package path, and a debug set so if someone wants to view the
 // corral create log
-func CreateCorral(ts *session.Session, corralName, packageName string, debug bool, cleanup bool) ([]byte, error) {
+func CreateCorralAndWait(ts *session.Session, corralName, packageName string, debug, cleanup bool) ([]byte, error) {
```

Comment:
What's the point of this func? It's doing the exact same thing as the other CreateCorral.

Reply:
createCorralAndWait is the old function, createCorral (if you look at shepherd / main)

CreateCorral does not use command().CombinedOutput (which waits on the job to finish, then returns) 
not using combinedOutput allows us to run corral in parallel, as now we don't have to wait on each job to finish in order to start the next one. 

see use of this here: https://github.com/rancher/shepherd/blob/782ae7375e7c01b7b9cc16d47e81b9cec54b6c84/extensions/provisioning/creates.go#L650-L655

Reply:
ah I see so you have removed CombinedOutput? 
Can you make this a separate PR? The provisioning has been moved anyways. Or you can wait until my work is done shouldn't be much longer.

Reply:
yes, CombinedOutput is removed from the CreateCorral call, it is now in the CreateCorralAndWait call